changelog/fix-icons

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: accessibility
+
+Updated featured icons in all calendar views to include accessible labels. Reorganized the Month view Feature icons to be more understandable when read by a screen reader.
```

Comment:
Is this the correct changelog entry?

Reply:
That's from other work on the base branch, I need to update